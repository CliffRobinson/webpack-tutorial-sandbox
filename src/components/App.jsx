import React from 'react'

import {ChatWindowContainer} from './ChatWindowContainer'
import {data} from '../../server/db/seeds/test/test-chat'

export function App() {
    return <div> 
        <p>I am rendered from React!!!!</p>
        <ChatWindowContainer />
    </div>
}