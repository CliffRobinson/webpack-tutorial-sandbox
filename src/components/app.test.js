import React from 'react';
import {shallow} from 'enzyme';

import {App} from './App';

describe('components/app/js', ()=> {
    it('matches snapshot', ()=> {
        const wrapper = shallow(<App />)
        expect(wrapper).toMatchSnapshot()
    })
})