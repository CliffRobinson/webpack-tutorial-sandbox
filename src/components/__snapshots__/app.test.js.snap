// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/app/js matches snapshot 1`] = `
<div>
  <p>
    I am rendered from React!!!!
  </p>
  <Connect(ChatWindow)
    room_id={0}
  />
</div>
`;
